src/Api/Endpoints/Project/Instrument.php

```diff
@@ -113,7 +113,7 @@ public function handle(ServerRequestInterface $request) : ResponseInterface
      *
      * @return array
      */
-    private function _toArray(ServerRequestInterface $request)
+    private function toArray(ServerRequestInterface $request)
```

Comment:
@xlecours can you verify that this won't break anything? I suspect that maybe this one should be protected too, but it's your code

Reply:
@driusan  `private` is correct